protos/route.proto

```diff
@@ -343,10 +345,29 @@ message InitialLoadSummaryResponse {
 message GetCDCBatchesRequest {
   string flow_job_name = 1;
   uint32 limit = 2;
+  bool ascending = 3;
+  int64 before_id = 4;
+  int64 after_id = 5;
 }
 
 message GetCDCBatchesResponse {
   repeated CDCBatch cdc_batches = 1;
+  int32 total = 2;
+  int32 page = 3;
+}
+
+message GraphRequest {
+  string flow_job_name = 1;
+  string aggregate_type = 2; // TODO name?
+}
+
+message GraphResponseItem {
+  double time = 1;
+  double rows = 2;
```

Comment:
`double` for rows?

Reply:
int64 in protos gets put into typescript as number, but then sent as string. Then we have to convert down to number anyways for graph component